strings.txt

```diff
@@ -285,10 +285,11 @@ PLUGIN_QOBUZ_BOOKLET
 	NL	Boekje
 
 PLUGIN_QOBUZ_PERFORMERS
-	DE	Performer
-	EN	Personnel
-	FR	Interprète
-	NL	Performer
+	DE	Credits
+	EN	Credits
+	FR	Crédits
+	NL	Credits
+	ES	Créditos
```

Comment:
BTW: you know that EN, FR, ES would have been good enough? If DE wasn't found it would use EN, which is identical in this case.